graphene/src/layers/text_layer/mod.rs

```diff
@@ -82,14 +90,24 @@ impl LayerData for TextLayer {
 		Some([(x0, y0).into(), (x1, y1).into()])
 	}
 
-	fn intersects_quad(&self, quad: Quad, path: &mut Vec<LayerId>, intersections: &mut Vec<Vec<LayerId>>) {
-		if intersect_quad_bez_path(quad, &self.bounding_box(&self.text).to_path(0.), true) {
+	fn intersects_quad(&self, quad: Quad, path: &mut Vec<LayerId>, intersections: &mut Vec<Vec<LayerId>>, font_cache: FontCache) {
+		let buzz_face = self.load_face(font_cache);
+
+		if intersect_quad_bez_path(quad, &self.bounding_box(&self.text, buzz_face).to_path(0.), true) {
 			intersections.push(path.clone());
 		}
 	}
 }
 
 impl TextLayer {
+	pub fn load_face<'a>(&self, font_cache: FontCache<'a>) -> Face<'a> {
+		const DEFAULT_FONT: &str = "http://fonts.gstatic.com/s/merriweather/v28/u-440qyriQwlOrhSvowK_l5OeyxNV-bnrw.ttf";
```

Comment:
I'm concerned about having to include this both here and in `Document.vue`, as well as the fact that I don't trust this URL to be valid forever in the future. I'd much prefer if we can preload the font list and then the default font (Merriweather). Does that seem feasible?

Reply:
You now just specify the font name and variant once in `fonts.ts` and it sets a rust property for the default font name.

Reply:
Splendid!